src/main/java/com/labuhn/minesweeper/domain/Field.java

```diff
@@ -0,0 +1,36 @@
+package com.labuhn.minesweeper.domain;
+
+
+import java.security.InvalidParameterException;
+
+public class Field {
+
+
+    Cell[][] cells;
+
+    public Field(int rowCount,int columnCount){
+        validate(rowCount,columnCount);
+        cells = new Cell[rowCount][columnCount];
+        initialize();
+    }
+
+    private void validate(int rowCount, int columnCount) {
+        if(rowCount <=0 || columnCount <= 0){
+            throw new InvalidParameterException(String.format("Initialized field with non " +
```

Comment:
I don't know if its really good to throw exceptions. The UI won't show any errors. Maybe the validation could return a boolean and the logic would just ignore the user input for invalid cases?